backend/resources/participant.py

```diff
@@ -37,7 +37,7 @@ def post(self):
         db.session.commit()
 
         return {"message": "Participant created successfully.",
-                "participant": data}, HTTPStatus.CREATED
+                "participant": participant_schema.dump(participant)}, HTTPStatus.CREATED
 
```

Comment:
Length of this line is ok, if we want to switch to black's default 88 characters. If we want to stick to PEP8 it is to long.

Reply:
I would stick with black. :)